### PR TITLE
tracing: minor usability improvements

### PR DESCRIPTION
**Pull request by jordanlewis**
- rename txn heartbeat trace to "heartbeat txn" from "heartbeat"
- rename "sql txn implicit" trace to "sql txn" with the "implicit=true" tag

### CODE REVIEW

File: pkg/sql/txn_state.go [hunk 1]
```diff
@@ -145,12 +145,7 @@ func (ts *txnState2) resetForNewSQLTxn(
 	// TODO(andrei): figure out how to close these spans on server shutdown? Ties
 	// into a larger discussion about how to drain SQL and rollback open txns.
 	var sp opentracing.Span
-	var opName string
-	if txnType == implicitTxn {
-		opName = sqlImplicitTxnName
-	} else {
-		opName = sqlTxnName
-	}
+	opName := sqlTxnName
```
Comment (nvanbenschoten) [inline pkg/sql/txn_state.go]: Is there a reason to keep this variable around?

**Review by nvb — APPROVED** [review]

**Comment by andreimatei** [comment]
:lgtm:

---

Review status: 0 of 3 files reviewed at latest revision, 1 unresolved discussion, some commit checks failed.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/cockroachdb/cockroach/22946#-:-L5yjYB_cD0wfi2sbPOh:bnfp4nl)*

File: pkg/sql/txn_state.go (hunk 1 above)
Comment (jordanlewis) [reply pkg/sql/txn_state.go]: Yes it's used several times below.

**Comment by jordanlewis** [comment]
TFTRs!

**Pull request merged**